tests/framework/test_util.cpp

```diff
@@ -57,15 +57,13 @@ void print_error_message(LSTATUS status, const char* function_name, std::string
     LocalFree(lpMsgBuf);
 }
 
-bool set_env_var(std::string const& name, std::string const& value) {
-    bool ret = SetEnvironmentVariableA(name.c_str(), value.c_str());
-    if (ret == false) {
+void set_env_var(std::string const& name, std::string const& value) {
+    BOOL ret = SetEnvironmentVariableA(name.c_str(), value.c_str());
```

Comment:
Won't this fail on Linux?

Reply:
This is the WIN32 part of the code, so it shouldn't be an issue. There is another definition of `set_env_var` further down for linux/other platforms.